modules/axtask/src/task.rs

```diff
@@ -182,6 +182,23 @@ impl TaskInner {
             None
         }
     }
+
+    /// Gets the cpu affinity mask of the task.
+    ///
+    /// Returns the cpu affinity mask of the task in type [`cpumask::CpuMask`].
+    #[inline]
+    pub fn cpumask(&self) -> CpuMask {
+        *self.cpumask.lock()
+    }
+
+    /// Sets the cpu affinity mask of the task.
+    ///
+    /// # Arguments
+    /// `cpumask` - The cpu affinity mask to be set in type [`cpumask::CpuMask`].
+    #[inline]
+    pub fn set_cpumask(&self, cpumask: CpuMask) {
+        *self.cpumask.lock() = cpumask
+    }
```

Comment:
Btw move `get_cyx_mut` and `kernel_stack_top` here, forgot about it before.

Reply:
ok